database_docs/doc_base.json

```diff
@@ -1,404 +1,404 @@
+    "minimum_dependent_variable_output": "Minimum dependent variable output",
+    "minimum_energy_efficiency_ratio": "Minimum energy efficiency ratio (EER)",
+    "minimum_energy_efficiency_ratio_2": "Minimum energy efficiency ratio (EER2)",
+    "minimum_heating_seasonal_performance_factor": "Minimum heating seasonal performance factor (HSPF)",
+    "minimum_heating_seasonal_performance_factor_2": "Minimum heating seasonal performance factor (HSPF2)",
+    "minimum_independent_variable_1": "Minimum value for the independent variable 1",
+    "minimum_independent_variable_2": "Minimum valiue for the independent variable 2",
+    "minimum_integrated_energy_efficiency_ratio": "Minimum integrated energy efficiency ratio (IEER)",
+    "minimum_integrated_part_load_value": "Minimum integrated part load value (IPLV)",
+    "minimum_kilowatts_per_tons": "Minimum kilowatts per tons",
+    "minimum_percent_of_surface": "Minimum percentage of surface area",
+    "minimum_performance_btu_per_hr_per_hp": "Minimum heat rejection per nominal horse power (Btu/h.hp)",
+    "minimum_performance_gpm_per_hp": "Minimum heat rejection gallon per minute per nominal horse power (gpm/hp)",
+    "minimum_projection_factor": "Minimum projection factor",
+    "minimum_scop": "Minimum sensible coefficient of performance (SCOP)",
+    "minimum_seasonal_energy_efficiency_ratio": "Minimum seasonal energy efficiency ratio (SEER)",
+    "minimum_seasonal_energy_efficiency_ratio_2": "Minimum seasonal energy efficiency ratio 2 (SEER2)",
+    "minimum_storage": "Minimum storage capacity (gal)",
+    "minimum_thermal_efficiency": "Minimum thermal efficency (Et)",
+    "minimum_water_cooled_chilled_water_capacity": "Minimum water cooled chilled water capacity (Btu/h)",
+    "minimum_water_cooled_chilled_water_capacity_no_fan_cooling": "Minimum water cooled chilled water capacity (Btu/h) without fan",
+    "name": "Name",
+    "natural_gas_equipment_average_epd": "Average natural gas equipment power density",
+    "natural_gas_equipment_epd_unit": "Natural gas equipment power density unit",
+    "natural_gas_equipment_fraction_latent": "Natural gas equipment power latent fraction",
+    "natural_gas_equipment_fraction_lost": "Natural gas equipment power lost fraction",
+    "natural_gas_equipment_fraction_radiant": "Natural gas equipment power radiant fraction",
+    "natural_gas_equipment_maximum_epd": "Maximum natural gas equipment power density",
+    "natural_gas_equipment_median_epd": "Median natural gas equipment power density",
+    "natural_gas_equipment_minimum_epd": "Minimum natural gas equipment power density",
+    "natural_gas_equipment_space_type_name": "Natural gas space type name",
+    "nominal_full_load_efficiency": "Minimum required motor full load efficiency",
+    "nontransient_dwelling": "Nontransient dwelling",
+    "number_of_poles": "Number of poles",
+    "occup_sensor_auto_on_svgs": "Occupancy sensor reduction",
+    "occup_sensor_savings": "@todo",
```

Comment:
@AmmarDeh to add a description.

Reply:
"occup_sensor_auto_on_svgs": "Occupancy sensor reduction for manual-ON or partial-auto-ON occupancy sensors",
    "occup_sensor_savings": "Occupancy sensor reduction",

Reply:
Thanks!